user/pages/03.commerce2/02.developer-guide/02.install-update/04.uninstalling/docs.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Uninstalling
+taxonomy:
+    category: docs
+---
+
+In order to completely remove Commerce from your Drupal site, you will first
+need to delete all existing data. You can do this through the Admin UI using
+the `Extend` toolbar item and then clicking on the `Uninstall` tab.
+
+Type 'commerce' into the text box at the top of the page to filter the list by
+Commerce modules.
+
+Scan through the list for items that say:
+```bash
+There is content for the entity type:
+```
+
+Click the provided link to permanently remove the content for each entity type.
+
+
+For example, if you have created a Store for your site, you will see:
+
+![uninstall_store](../images/04.uninstall-store.jpg)
+
+Click the `Remove stores` link.
+
+
+Once all content has been removed, you can start uninstalling the commerce
+modules. You will only be able to uninstall a few at a time because of
+dependencies. For example, you will not be able to uninstall the base Commerce
+module until all other commerce modules are installed. Note: if you have
```

Comment:
This is missing the "run cron" trick documented in https://www.drupal.org/project/commerce/issues/2791303